pom.xml

```diff
@@ -52,31 +52,36 @@
 			<url>http://imagej.net/User:Leon</url>
 			<properties><id>lnyng</id></properties>
 		</contributor>
+		<contributor>
```

Comment:
I'd suggest adding yourself as `<developer>` and would even suggest handing over more responsibility to you, since I won't have too much time to actively work on this. If not, that's also fine 🙂

Reply:
Agreed, I can take over (at least in part) the responsibility for this component. I noticed that, although I'm "owner" in the `scijava` org, I don't have permissions to push on this specific repository, presumably because it was moved with the permissions it had before... Can you make me an admin on this one?

Reply:
Uhm. You should at least have write access to this repository now via your *Hackers* membership. I have, however, managed to screw up the rights on this repository even more by adding the *Hackers* group to it: it seems that @ctrueden is - by some black magic (i.e. reasons that are not known to me) - the only one with admin access to this repository left.

Reply:
Right: I don't see the *Settings* tab on this repository, but I do have push permissions and can merge this PR. So I'll go ahead shortly. Thanks!

Reply:
@imagejan Any chance you were using the mobile version of GitHub? I did not see a Settings tab either in that mode. But switching to desktop mode, it was there. In any case, I added you to this repository as a collaborator with admin access.

Reply:
I was always using the desktop mode, if I'm not mistaken. But now I also see the Settings tab, so changes were effective. Thanks a lot, @ctrueden and @stelfrich!